modules/regex/tests/test_regex.h

```diff
@@ -131,6 +131,48 @@ TEST_CASE("[RegEx] Empty Pattern") {
 	CHECK(re.is_valid());
 }
 
+TEST_CASE("[RegEx] Complex Grouping") {
+	const String test = "https://docs.godotengine.org/en/latest/contributing/";
+
+	// Ignored protocol in grouping
```

Comment:
```suggestion
	// Ignored protocol in grouping.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide).